AnkiDroid/src/main/java/com/ichi2/libanki/stats/Stats.java

```diff
@@ -226,7 +226,7 @@ private String getRevlogTimeFilter(AxisType timespan, boolean inverse) {
         long cut = mCol.getSched().getDayCutoff();
         int cardCount = 0;
         String query = "select " +
-                "(cast((id/1000.0 - ?) / 86400.0 as int))/? as day," +
+                "(cast((id/1000.0 - ?) / SECONDS_PER_DAY.0 as int))/? as day," +
```

Comment:
this can't be correct?
```suggestion
                "(cast((id/1000.0 - ?) / SECONDS_PER_DAY as int))/? as day," +
```

Reply:
Clearly not.